docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think we were supposed to only have the AxonIQ license in here?

Reply:
If we have any file which is under the Apache license in there, we need to add it.

Reply:
Fair, that makes sense.
Do you also know whether there is a file under the Apache license in axon-server-se?

Reply:
@bert-laverman, @MGathier do you know if we have any files under the Apache license? if not I can remove it...

Reply:
As mentioned on Slack, the Roboto slab fonts are under the Apache 2.0 license